source/main.cc

```diff
@@ -631,12 +649,30 @@ int main (int argc, char *argv[])
                                     "different space dimension is given in the parameter file."));
         }
     }
+  catch (ExceptionBase &exc)
```

Comment:
Identical except ``exc.get_exc_name() `` which is not reported in ``exc.what()`` for whatever reason.